### ISSUE TITLE
Suggestion: Remove Overwolf and release portable version again

### ISSUE THREAD

**Issue by RiQuY** [Kyusung4698/PoE-Overlay#741]
Remove Overwolf and release portable version again.

No one asked for that and Overwolf is known for being buggy in many games.

Thanks.

**Comment by NyvenZA** [comment]
I agree. +1 
Tried to install the Overwolf version. 
* Framerate immediately drops from 60FPS to 30FPS in game. 
* POE Overlay does not start up when selecting it from the Overwolf menu. 
* POE Overlay doesn't start if I manually try to start it.
* Hotkeys do not work and the OverWolf doesn't even open when I press CTRL+TAB. 
* Lots of new processes that I don't want. 

At this rate I am going back to the Auto Hot key scripts that works.

**Comment by conetopia** [comment]
I agree with the idea of a portable/simple-limited version to just check prices, and a full feature Overwolf edition version, that want all new cool Features like the in-game market.
Some players pc's can't handle it ( especially if running on an old school Cheap hard drive )
Takes a little longer to load, 20 vs 30 sec for the avg user.

Poe_overlay + overwolf is not resource crazy.
If your FPS drops from 60 to 30 ,  you probably have a buget pc, and maxed % on cpu/ram/storage ect. Need to buy a used upgrade, or lower your desktop graphic settings ect.
a couple of new processes that use like almost no resources... not a big deal.

**Comment by HankTheDrunk** [comment]
I simply won't use this if it requires overwolf

I'm sure the devs don't care, but it's not just me. I'm just loud enough to post about it.

have fun losing users

**Comment by Taffa2** [comment]
+1 

I also will never use Overwolf and install this app.

Please bring back the stand-alone version.

REALLY crappy decision.

**Comment by darkwiz666** [comment]
Just wanted to use the overlay and Overwolf won't even install on my system. :/
Portable version never had this issue...

**Comment by RiQuY** [comment]
I know the dev checks all the issues and answers them.

So can you please stop ignoring this one and give us a answer, even if the answer is negative.

Thanks.

**Comment by ParsHere** [comment]
Yeah, there's a reason he's silent about this. Fortunately there is an overwolf-free fork available. Just search PoE reddit.

**Comment by MetalJacx** [comment]
I also agree with the removal of Overwolf, for future releases. Overwolf is trash and won't use this because of Overwolf requirement. I went open source for wow addons to when they force that change.

**Comment by ganego** [comment]
The answer is simple. They pay him money, so no he won't change nor answer.
